src/domain/communication/updates/CommunityUpdatesContainer/CommunityUpdatesContainer.tsx

```diff
@@ -84,11 +82,11 @@ export const CommunityUpdatesContainer: FC<CommunityUpdatesContainerProps> = ({
       }
       const update = await removeUpdate({
         variables: { messageData: { messageID, roomID } },
-        refetchQueries: spaceId && communityId ? [refetchCommunityUpdatesQuery({ communityId })] : [],
+        refetchQueries: communityId ? [refetchCommunityUpdatesQuery({ communityId })] : [],
```

Comment:
What I'm putting here when I have a case like this is `[refetchCommunityUpdatesQuery({ communityId: communityId! })]`